### ISSUE TITLE
Merge report of each module to one report

### ISSUE THREAD

**Issue by YouSsef3Ssam** [vanniktech/gradle-android-junit-jacoco-plugin#211]
When I run `./gradlew mergeJacocoReports` it fails as a missing task.

```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'mergeJacocoReports' not found in root project
```

**Comment by 0neel** [comment]
This repo seems to be abandoned. We recently migrated to https://github.com/NeoTech-Software/Android-Root-Coverage-Plugin which supports AGP 7.2 and has some cool features simplifying its configuration.

**Comment by vanniktech** [comment]
Yep https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/issues/202#issuecomment-930607016
